src/index.html

```diff
@@ -7,6 +7,70 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--month-length--31">
+        <div class="calendar__day calendar__day--start-day--mon"></div>
```

Comment:
```suggestion
      <div class="calendar calendar--month-length--31 calendar--start-day--mon">
        <div class="calendar__day"></div>
```
for start day it should be also modifier for calendar block
making margins for first day you can use :first-child pseudo class